frontend/js/recordings.js

```diff
@@ -146,3 +170,27 @@ function mapWordformsToBestRecordingURL(response) {
 
   return wordform2recordingURL;
 }
+
+/**
+ * Chunks the iterable into subarrays of a maximum size.
+
+ * @param {Iterable<T>}  one or more wordforms to search for.
+ * @return {Array<Array<T>}}
+ */
+function chunk(collection) {
```

Comment:
This took me a while to mentally trace through, but ultimately it makes sense. This replaces the `lodash` package that we imported, right?

Reply:
Correct! I decided I could do it with a tiny function instead of importing a package. Would adding loop invariants help to understanding the code?

Reply:
Maybe? I found following `currentChunk` was the confusing part. I couldn't understand how it wound up with anything in it if it was being reset to an empty list, but after properly thinking about it, it made sense.